ScriptLoader/ScriptLoader.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace ScriptLoader
 {
-    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "1.2.4")]
+    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "1.2.5")]
```

Comment:
I prefer to not bump version numbers from PRs, also this is a breaking change so the version should probably be changed to 2.0.